rlogin/src/components/GetBalance.jsx

```diff
@@ -0,0 +1,30 @@
+import { useContext, useState } from 'react';
+import rLoginContext from '../contexts/rLoginContext';
+
+function GetBalance() {
+  const { provider, account, tryRequest } = useContext(rLoginContext);
+
+  const [balance, setBalance] = useState('');
+
+  const getBalance = () => {
+    tryRequest('Requesting balance', async () => {
+      const weiBalance = await provider.request({
+        method: 'eth_getBalance',
+        params: [account],
+      });
+      // convert hexidecimal wei value to RBTC
+      const rbtcBalance = parseInt(weiBalance, 16) / 10 ** 18;
```

Comment:
this method may not work for large values, where wei balance is more than Javascript's max integer (`uint256` from solidity vs `Number` from js) ... instead use a util function like `BigNumber` for this